Front-End/src/Pages.js

```diff
@@ -447,17 +398,12 @@ function Intake(props) {
                         <option value="ONYX">ONYX</option>
                     </select>
                     <br></br>
-                    <input type="text" id="LengthInInches" name="LengthInInches" placeholder="Length in Inches" className="text-input" />
+                    <input type="text" id="LengthInInches" name="LengthInInches" placeholder="Length in Inches" className="text-input" required="required" />
                     <br></br>
-                    <input type="text" id="PartNumber" name="PartNumber" placeholder="Part Number" className="text-input" />
+                    <input type="text" id="PartNumber" name="PartNumber" placeholder="Part Number" className="text-input" required="required" />
                     <br></br>
                     <h4>When do you need the part done by?</h4>
                     <NewDate />
```

Comment:
I'm not sure how we make this required, but we will probably want to do it eventually. We can ignore this one for now, but this could be a good thing to look into